commands/sync.go

```diff
@@ -28,6 +34,15 @@ var SyncCmd = &cli.Command{
 var SyncStatusCmd = &cli.Command{
 	Name:  "status",
 	Usage: "Report sync status of a running lily daemon",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:     "output",
+			Usage:    "Print only the current sync stage at the latest height. One of [text, json]",
+			Aliases:  []string{"o"},
+			Required: false,
+			Value:    "text",
```

Comment:
When the flag isn't provided its default value will be `"text"`, is this intended? In order for this [case](https://github.com/filecoin-project/lily/pull/1010/files#diff-ab57c1124da90ef9a1f72210e299ca8a5facfb7a36b704dcbeb803957c6b9525R91) to evaluate the user will need to explicitly set the flags value to `""`.

Reply:
good catch, thanks